privacy_evaluator/utils/trainer.py

```diff
@@ -32,7 +30,6 @@ def trainer(
     elif isinstance(model, nn.Module):
```

Comment:
It seems like you are lacking an else case here. And assertion if its none of the types you can cater.

Reply:
We added an else case which raises an type error.